src/components/templates/Header.jsx

```diff
@@ -26,40 +26,33 @@ const Header = () => {
   }, []);
 
   return (
-    <header className="sticky">
-      <Divider />
+    <header className="bg-secondary sticky left-0 right-0 top-0 z-20">
       <Container>
-        <div className="sm:flex sm:items-center sm:justify-between">
-          <div className="relative flex items-center justify-between">
-            <IconItem
-              src="/images/hooBank.svg"
-              alt="hooBank"
-              title_part1="Hoo"
-              title_part2="Bank"
+        <div className="flex items-center py-4">
+          <Link href={"/"}>
+            <LogoItem
```

Comment:
If the logo icon is different then pass the svgPath prop, otherwise directly add it to the Logo.